javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,45 @@
+const width = 4;
+const height = 5;
+
+const area = width * height;
+console.log('value of area:', area);
+console.log('typeof area:', typeof area);
+
+const bill = 15;
+const payment = 20;
+
+const change = payment - bill;
+console.log('value of change:', change);
+console.log('typeof change:', typeof change);
+
+const quizzes = 95;
+const midterm = 90;
+const final = 85;
+
+const grade = (quizzes + midterm + final) / 3;
+console.log('value of grade:', grade);
+console.log('typeof grade:', typeof grade);
+
+const firstName = 'Tim';
+const lastName = 'Woo-Sam';
+
+const fullName = firstName + ' ' + lastName;
+console.log('value of fullName:', fullName);
+console.log('typeof fullName:', typeof fullName);
+
+const pH = 2;
+
+const isAcidic = pH < 7;
+console.log('value of isAcidic:', isAcidic);
+console.log('typeof isAcidic:', typeof isAcidic);
+
+const headCount = 100;
+
+const isSparta = headCount === 300;
+console.log('value of isSparta:', isSparta);
+console.log('typeof isSparta:', typeof isSparta);
+
+let motto = fullName;
+motto = fullName + ' ' + 'is the GOAT';
```

Comment:
You should be using the `+=` operator here rather than just the normal `=` operator